token-lending/sdk/src/state/lending_market.rs

```diff
@@ -62,6 +65,8 @@ pub struct InitLendingMarketParams {
     pub oracle_program_id: Pubkey,
     /// Oracle (Switchboard) program id
     pub switchboard_oracle_program_id: Pubkey,
+    /// Current slot
+    pub current_slot: u64,
```

Comment:
Why was this added?

Reply:
ah, i used to call RateLimiter::new() in init_lending_market, which needed the current slot. but then later i added a Default implementation. will remove